guides/posts/rapidapi-teams/post.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Share and Collaborate on APIs using RapidAPI for Teams
+slug: rapidapi-teams
+description: If your team consumes, develops, or manages APIs, try Rapid API Teams. Today, more than 60,000 teams are using it for collaborating on APIs.
+publishedDate: 2021-10-02T19:10:30.765Z
+lastModifiedDate: 2021-10-02T19:10:30.765Z
+authors:
+  - bilal
+category: api
+tags:
+  - api
+  - teams
+  - collaboration
+coverImage: ''
+draft: false
+---
+
+<Lead>
+If your team consumes, develops, or manages APIs, then consider Rapid API Teams. Today, more than 60,000 teams are using it for collaborating on APIs.
+</Lead>
+
+It lets you share, publish and collaborate on your APIs in a private workspace.
+
+## Private Workspace
+
+As companies use more and more APIs, it gets difficult for the developers to keep track of them. Sharing the internal APIs with multiple developers across the organization becomes a challenge. Imagine this scenario; your team needs an external paid API. Let’s say you subscribe to the said API. Now, you would want a way to share it with your team so that all members can access it.
+
+RapidAPI Teams provides the collaborative solution you need for this purpose. You can make an organization on RapidAPI, invite your team members and start working on internal and external APIs without any hassle.
+
+Here is how Rapid API for Teams facilitates you in the consumption, publishing, and governance of APIs.
+
+## Features
+
+Here is how Rapid API for Teams facilitates you in the consumption, publishing, and governance of APIs.
+
+### Consumption
+
+For starters, the private workspace allows you to search for an API in your organization or even look up a specific API endpoint. The organization has a developer dashboard that shows detailed analytics like latencies, costs, requests per endpoint, and consumption of the subscribed APIs. These are all super helpful features.
+
+### Publishing
+
+RapidAPI provides its familiar provider interface with added features for publishing APIs in the organization. They can be public or private, and you can choose the pricing plans according to your needs. From a single dashboard, you can manage the monetization and usage of your published APIs.
+
+### Governance
+
+Whether your team is selling APIs or consuming them, the private workspace allows you to manage them conveniently through one dashboard. Access control allows you to give or restrict API access to members. You can also set custom plans that limit access to specific endpoints.
+
+## Create a new organization on RapidAPI Teams
+
+Enough talk, let’s go ahead and create our new organization in RapidAPI Teams. Navigate to the [Teams page](https://rapidapi.com/products/teams/?utm_source=rapidapi.com/guides&utm_medium=DevRel&utm_campaign=DevRel). [Create an account](https://RapidAPI.com/auth/sign-up?referral=/hub?utm_source=rapidapi.com/guides&utm_medium=DevRel&utm_campaign=DevRel) if you haven’t already.
```

Comment:
The correct parameter is **RapidAPI.com/guides&utm_medium=DevRel&utm_campaign=DevRel**.